ios/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## 2017-07-11 — LinkKit 1.0.6
+### Added
+- Add [copy customization](https://blog.plaid.com/link-copy-customization/) which allows to change the text of certain user interface elements in the Link flow
+- Add exit button when searching for an institution yielded no results so people can directly exit out of Link iOS
```

Comment:
What do you think about using copy from the plaid.com/docs/api#onexit-callback :

```
- Add an exit button, to handle the situation when a user exited the Link flow after unsuccessfully (no results returned) searching for a financial institution
```